Cargo.toml

```diff
@@ -17,6 +17,7 @@ build = "build.rs"
 libc = "0.2"
 nom = { version = "2", features = ["verbose-errors"] }
 byteorder = "0.5"
+time = "0.1"
```

Comment:
It looks like the only thing getting used from time is Duration, could we use std::time::Duration instead?